conf/app.routes

```diff
@@ -442,3 +442,9 @@ GET        /:taxYear/uk-rent-a-room/expenses/legal-management-other-fee
 POST       /:taxYear/uk-rent-a-room/expenses/legal-management-other-fee                  controllers.ukrentaroom.LegalManagementOtherFeeController.onSubmit(taxYear: Int, mode: Mode = NormalMode)
 GET        /:taxYear/uk-rent-a-room/expenses/change-legal-management-other-fee                        controllers.ukrentaroom.LegalManagementOtherFeeController.onPageLoad(taxYear: Int, mode: Mode = CheckMode)
 POST       /:taxYear/uk-rent-a-room/expenses/change-legal-management-other-fee                        controllers.ukrentaroom.LegalManagementOtherFeeController.onSubmit(taxYear: Int, mode: Mode = CheckMode)
+
+GET        /:taxYear/uk-rent-a-room/other-property-rr-expenses                       controllers.OtherPropertyRRExpensesController.onPageLoad(taxYear: Int, mode: Mode = NormalMode)
```

Comment:
To make it consistent:  /:taxYear/uk-rent-a-room/expenses/other-allowable-property-expenses

Reply:
Fixed with latest commit